e2e/modern-fake-timers/from-config/__tests__/test.js

```diff
@@ -16,3 +16,13 @@ test('fake timers', () => {
 
   expect(Date.now()).toBe(1000);
 });
+
+test('fake timers with Date argument', () => {
+  jest.setSystemTime(new Date(0));
+
+  expect(Date.now()).toBe(0);
+
+  jest.setSystemTime(1000);
```

Comment:
this doesn't use a `Date`, but a number like above

Reply:
🤦 it does indeed, thank you.